crates/containerd-shim-wasm/src/sandbox/shim/cli.rs

```diff
@@ -70,9 +69,6 @@ where
             .and_then(|a| a.get("io.kubernetes.cri.sandbox-id"))
             .unwrap_or(&id);
 
-        setup_namespaces(&spec)
```

Comment:
Let's not make this change in this PR and keep it to https://github.com/containerd/runwasi/pull/503

Reply:
sounds good